examples/obj.rb

```diff
@@ -21,7 +21,7 @@ def to_s
   end
 end
 
-obj = Classy.new(23, ['abc', {x: true}])
+obj = Classy.new(23, ['abc', { x: true }])
```

Comment:
Perfect example of a mix of spaces after brackets and no space. I prefer consistent behaviour between `[]` and `{}`. Happy to go with either but not a mix.

Reply:
Maybe tight, no spaces after `[` or `{` or before `]` or `}` is best if there is no overriding consensus.